doozer/doozerlib/cli/images_streams.py

```
@@ -950,21 +950,49 @@ def reconcile_jira_issues(runtime, pr_map: Dict[str, Tuple[PullRequest.PullReque
+        found_issue = None
+        title_issue_keys = re.findall(r'([A-Z][A-Z0-9]+-\d+)', pr.title)
+        for candidate_key in title_issue_keys:
+            if not candidate_key.startswith(f'{project}-'):
+                continue
+            try:
+                candidate_issue = fetch_issue(candidate_key)
+                if candidate_issue.fields.summary in (summary, old_summary_format):
+                    found_issue = candidate_issue
+                    break
+            except Exception:
+                continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)